src/exception/exception-service.ts

```diff
@@ -172,6 +172,15 @@ export namespace operations {
     }
   };
 
+  export const getEntityException: Service = async ({ programId }) => {
+    const result = await entityExceptionRepository.find(programId);
+
+    return processResult({
+      result,
+      errorMessage: `no entity level exceptions for program '${programId}'`,
```

Comment:
I'm a little thrown off with this errorMessage pattern, it looks like we're passing in our result and then this will be the default value if it's empty?

Reply:
b/c of isRepo Error I see ... I feel like this message should be returned from isRepoError given an empty result, not passed in as a default, but it's not crucial, just opinion

Reply:
it's not a default though. the "repo error" is generic error for both program and entity exceptions.
`errorMessage` param is to have a more specific error message returned to user